src/cljc/ataru/translations/texts.cljc

```diff
@@ -640,196 +640,196 @@
-                                                 :sv "SV: Vahvistussähköposti lähetetty hakijalle"
+                                                 :sv "E-post med bekräftelse har skickats till sökande"
                                                  :en "EN: Vahvistussähköposti lähetetty hakijalle"}
    :contains-fields                             {:fi "Sisältää kentät:"
-                                                 :sv "SV: Sisältää kentät:"
+                                                 :sv "Innehåller fälten:"
                                                  :en "EN: Sisältää kentät:"}
    :copy-form                                   {:fi "Kopioi lomake"
                                                  :sv "Kopiera blanketten"
                                                  :en "EN: Kopioi lomake"}
    :copy-answer-id                              {:fi "Kopioi vastauksen tunniste leikepöydälle"
-                                                 :sv "SV: Kopioi vastauksen tunniste leikepöydälle"
+                                                 :sv "Kopiera svarstaggen till klippbordet"
                                                  :en "EN: Kopioi vastauksen tunniste leikepöydälle"}
    :copy-question-id                            {:fi "Kopioi kysymyksen tunniste leikepöydälle"
-                                                 :sv "SV: Kopioi kysymyksen tunniste leikepöydälle"
+                                                 :sv "Kopiera svarstaggen till klippbordet"
                                                  :en "EN: Kopioi kysymyksen tunniste leikepöydälle"}
    :created-by                                  {:fi "Luonut"
-                                                 :sv "SV: Luonut"
+                                                 :sv "Grundad av"
                                                  :en "EN: Luonut"}
    :custom-choice-label                         {:fi "Omat vastausvaihtoehdot"
-                                                 :sv "SV: Omat vastausvaihtoehdot"
+                                                 :sv "Egna svarsalternativ"
                                                  :en "EN: Omat vastausvaihtoehdot"}
    :decimals                                    {:fi "desimaalia"
-                                                 :sv "SV: decimaler"
+                                                 :sv "decimaler"
                                                  :en "EN: decimals"}
    :delete-form                                 {:fi "Poista lomake"
-                                                 :sv "SV: Ta bort blanketten"
+                                                 :sv "Ta bort blanketten"
                                                  :en "EN: Poista lomake"}
    :did                                         {:fi "teki"
-                                                 :sv "SV: teki"
+                                                 :sv "gjort av"
```

Comment:
Ei taida mennä ihan oikein, kun näitä käytetään muodossa `:did N :changes` eli esim. "Virkailijannimi teki 12 muutosta"? Ruotsiksi nyt "Virkailijannimi gjort av 12 ändringar"  Tämä on tietty erityisen hämäävää kun lausefragmentteja on ripoteltu ympäri lokalisaatiotiedostoa...

Reply:
Joo, ei mennyt oikein, korjaan tuonkin. Tosi on, ettei näistä aina saa selvää yhteys... Hyvä, että huomasit nämä!